maestro-cli/src/main/java/maestro/cli/util/DeviceConfig.kt

```diff
@@ -0,0 +1,32 @@
+package maestro.cli.util
+
+internal object DeviceConfigIos {
+
+    val device: String = "iPhone-11"
+    val versions = listOf(15, 16)
+    val runtimes = mapOf(
+        15 to "iOS-15-5",
+        16 to "iOS-16-2"
+    )
```

Comment:
Should we fetch this from Cloud instead? If not old CLI clients will be using the wrong minor version if we decide to move Cloud to for example iOS 16.4.

Reply:
I'm ok with it being hardocded for the time being. Minor versions usually don't introduce major OS changes and if it is an issue for a user then we can push them to update (provided we support it).

Open to discuss introducing remote config though.